src/accessibility.md.njk

```diff
@@ -52,8 +52,9 @@ The content listed below is non-accessible for the following reasons.
 
 The Design System website at [https://design-system.service.gov.uk/](https://design-system.service.gov.uk/) is partially compliant due to the following non-compliances:
 
-- users are not always notified when conditionally revealed content associated with a radio button or checkbox is expanded or collapsed - this fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
 - the section headings in accordions can be mistaken for links, but are treated as buttons - this fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)
+- on mobile, assistive technologies might interpret our guidance's HTML and Nunjucks tabs as plain text because the tabs are not contained by a parent element - this fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
```

Comment:
Thanks for picking this up so quickly, and a very neat and clear adaptation of some of the content on https://web.dev/aria-required-parent/ to explain the problem it causes for users 🙌🏻 

(I hadn't actually read that section properly, so I learned something from reviewing this!)

I also think it's nice and clear this is in the guidance rather than a component that might be used in a service. This accessibility statement is sort of trying to cover two things, so I wonder if there's a better way to structure this – but think that's probably a future thing to look at.

I would suggest a slight tweak, because the tabs definitely do have 'parents' – the important bit is that none of its parents are a tab list.

```suggestion
- on mobile, assistive technologies might interpret our guidance's HTML and Nunjucks tabs as plain text because the tabs are not contained within a tab list - this fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
```

Reply:
@36degrees Thanks! Your suggestion looks good. Will commit and send this update for pre-i.